arm-sql/2014-04-01/swagger/capabilities.json

```diff
@@ -0,0 +1,260 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Azure SQL Database capabilities",
+        "description": "Describes the Azure SQL capabilities available to a subscription for a given location.",
+        "version": "2014-04-01"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Sql/locations/{locationId}/capabilities": {
+            "get": {
+                "tags": [
+                    "Capabilities"
+                ],
+                "operationId": "Capabilities_List",
```

Comment:
According to https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-good-patterns.md section "Consistency in the naming of list methods", this should be `Capabilities_ListByLocation`.